### PR TITLE
fix: paste the data with filtered data

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* related issue(#1373)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: packages/toast-ui.grid/src/query/data.ts [hunk 1]
```diff
@@ -36,9 +36,21 @@ export function getCellAddressByIndex(
...
 }
 
-export function isEditableCell(data: Data, column: Column, rowIndex: number, columnName: string) {
-  const { disabled, editable } = data.filteredViewData[rowIndex].valueMap[columnName];
-  return !isHiddenColumn(column, columnName) && editable && !disabled;
+export function isEditableCell(store: Store, rowIndex: number, columnName: string) {
+  const { data, column } = store;
+  const { filteredIndex, filteredViewData } = data;
+
+  if (!filteredIndex || !isNil(filteredIndex[rowIndex])) {
```
Comment (adhrinae) [inline packages/toast-ui.grid/src/query/data.ts:43]: 조건문을 뒤집어서 early return을 시키는건 별로일까요?

```js
if (filteredIndex && isNil(filteredIndex[rowIndx]))) {
  return false;
}

// get index based on whole data(not filtered data)
const index = filteredIndex ? filteredIndex[rowIndex] : rowIndex;
makeObservable(store, index, true);

const { disabled, editable } = filteredViewData[rowIndex].valueMap[columnName];

return !isHiddenColumn(column, columnName) && editable && !disabled;
```

**Review by adhrinae — COMMENTED** [review]
리뷰 완료합니다.

**Review by seonim-ryu — APPROVED** [review]
리뷰 완료합니다. 고생하셨어요~

**Pull request merged**